application/app/src/test/java/uk/nhs/digital/uec/api/integration/fuzzysearch/FuzzySearchValidationErrorsTest.java

```diff
@@ -0,0 +1,102 @@
+package uk.nhs.digital.uec.api.integration.fuzzysearch;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.web.util.UriComponentsBuilder;
+import uk.nhs.digital.uec.api.model.ApiValidationErrorResponse;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+@ActiveProfiles({"prod"})
+/**
+ * Test class which passes requests through the Fuzzy Search endpoint and asserts desired API
+ * behavior. Only the model layer will be mocked here.
+ */
+public class FuzzySearchValidationErrorsTest {
+
+  @Autowired private ObjectMapper mapper;
+
+  private static String endpointUrl =
```

Comment:
Same comment as for the `FuzzySearchGeneralErrorsTest.java`

Reply:
Agreed